### PR TITLE
Update setup-julia for automerge

### PR DESCRIPTION
**Pull request by christopher-dG**

### CODE REVIEW

**Review by giordano — APPROVED** [review]

**Comment by christopher-dG** [comment]
I don't understand the [CI failure](https://github.com/JuliaRegistries/General/runs/1408903052?check_suite_focus=true#step:3:6).

**Comment by giordano** [comment]
Ouch

**Comment by giordano** [comment]
@SaschaMann

**Comment by DilumAluthge** [comment]
@christopher-dG Can you also update the remember_to_update_registryci.yml workflow file?

File: .github/workflows/automerge.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ jobs:
           - x64
     steps:
       - uses: actions/checkout@af513c7a016048ae468971c52ed77d9562c7c819 # v1.0.0
-      - uses: julia-actions/setup-julia@4469f5b9da960ff174efa528fea5889f8e873609 # v1.2.2
+      - uses: julia-actions/setup-julia@405bfcbfbd71503af238843c921b4a623e815267 # v1.4.1
```
Comment (giordano) [inline .github/workflows/automerge.yml:21]: Maybe try with an older version?
```suggestion
      - uses: julia-actions/setup-julia@d26d1111976eae5f00db04f0515ab744ec9cd79e # v1.3.1
```

Comment (DilumAluthge) [reply .github/workflows/automerge.yml:21]: Let's try it

**Comment by SaschaMann** [comment]
1.4.1 should be https://github.com/julia-actions/setup-julia/commit/d3ce119a16594ea9e5d7974813970c73b6ab9e94

The released builds are on separate release branches. The commit message is a bit misleading, sorry!

File: .github/workflows/automerge.yml [hunk 2]
```diff
@@ -18,7 +18,7 @@ jobs:
           - x64
     steps:
       - uses: actions/checkout@af513c7a016048ae468971c52ed77d9562c7c819 # v1.0.0
-      - uses: julia-actions/setup-julia@4469f5b9da960ff174efa528fea5889f8e873609 # v1.2.2
+      - uses: julia-actions/setup-julia@d26d1111976eae5f00db04f0515ab744ec9cd79e # v1.3.1
```
Comment (SaschaMann) [inline .github/workflows/automerge.yml:21]:
```suggestion
      - uses: julia-actions/setup-julia@d3ce119a16594ea9e5d7974813970c73b6ab9e94# v1.4.1
```

Comment (giordano) [inline .github/workflows/automerge.yml:21]: I took that hash from the [README](https://github.com/julia-actions/setup-julia/blob/405bfcbfbd71503af238843c921b4a623e815267/README.md#versioning)...
```suggestion
      - uses: julia-actions/setup-julia@3dcc911681c8af6e64fdc847e77c5e98f9d1611e # v1.3.1
```

Comment (SaschaMann) [reply .github/workflows/automerge.yml]: So I have no idea where I got `d26d1111976eae5f00db04f0515ab744ec9cd79e` from, it doesn't appear to be a valid commit.

File: .github/workflows/automerge.yml [hunk 3]
```diff
@@ -18,7 +18,7 @@ jobs:
           - x64
     steps:
       - uses: actions/checkout@af513c7a016048ae468971c52ed77d9562c7c819 # v1.0.0
-      - uses: julia-actions/setup-julia@4469f5b9da960ff174efa528fea5889f8e873609 # v1.2.2
+      - uses: julia-actions/setup-julia@3dcc911681c8af6e64fdc847e77c5e98f9d1611e # v1.3.1
```
Comment (giordano) [inline .github/workflows/automerge.yml:21]:
```suggestion
      - uses: julia-actions/setup-julia@d3ce119a16594ea9e5d7974813970c73b6ab9e94# v1.4.1
```

File: .github/workflows/automerge.yml [hunk 4]
```diff
@@ -18,7 +18,7 @@ jobs:
           - x64
     steps:
       - uses: actions/checkout@af513c7a016048ae468971c52ed77d9562c7c819 # v1.0.0
-      - uses: julia-actions/setup-julia@4469f5b9da960ff174efa528fea5889f8e873609 # v1.2.2
+      - uses: julia-actions/setup-julia@d3ce119a16594ea9e5d7974813970c73b6ab9e94# v1.4.1
```
Comment (giordano) [inline .github/workflows/automerge.yml:21]:
```suggestion
      - uses: julia-actions/setup-julia@d3ce119a16594ea9e5d7974813970c73b6ab9e94 # v1.4.1
```

**Comment by christopher-dG** [comment]
I think this is good now.

**Pull request merged**